src/browser/assistant/history.js

```diff
@@ -61,18 +61,14 @@ const pendingRequests = Object.create(null);
 
 const abort =
   (id/*:number*/)/*:Task<Never, number>*/ =>
-  Task.future(() => new Promise(resolve => {
-
-  }));
+  new Task(succeed => void(0))
```

Comment:
Using `void(0)` used to be better than `undefined` back when `undefined` was configurable so could be changed to something else. That hasn't been the case for many years now, so `undefined` isn't the least bit worse. (Except for being 2 chars longer, which is why minifiers still use it.)